apps/calendar/js/views/event_base.js

```diff
@@ -316,7 +268,7 @@ EventBase.prototype = {
     if (id) {
       classList.add(this.UPDATE);
 
-      this._loadModel(id, completeDispatch);
+      this._loadModel(id).then(completeDispatch);
```

Comment:
the modify/view event views deserve a complete refactor in the future.. but for now I think it's good enough.